index.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
     require('babel-plugin-lodash'),
     require('babel-plugin-transform-export-extensions'),
     require('babel-plugin-transform-object-rest-spread'),
+    require('@babel/plugin-proposal-class-properties'),
```

Comment:
Change this to `babel-plugin-transform-class-properties`!

Reply:
oh dang. Forgot there was another spot 😬